packages/react-components/src/components/styles/MessageThread.styles.ts

```diff
@@ -31,7 +31,8 @@ export const messageThreadContainerStyle = mergeStyles({
   maxHeight: '100%',
   overflow: 'auto',
   position: 'relative',
-  alignSelf: 'center'
+  alignSelf: 'center',
+  minWidth: '320px'
```

Comment:
Our current minWidth for the chat composite is 280px:
![image](https://user-images.githubusercontent.com/2684369/163074968-6e626f4f-e7a2-4755-a2c9-61b3ee8aea7c.png)
https://60c7ae6891f0e90039d7cd54-actiqxaqcq.chromatic.com/?path=/docs/composites-chat-basicexample--basic-example

Can we reduce the space the file cards need?

Reply:
@pereiralex thoughts?

Reply:
This dimension came from AlexP.
@prabhjot-msft let's reach out to AlexP and get his opinion here.

Reply:
note: for context this was reduced to support the galaxy fold